server/js/sockets.js

```diff
@@ -97,6 +97,7 @@ sockets.init = function socketInit(server) {
   mqttClient.subscribe(DAS.start);
   mqttClient.subscribe(DAS.stop);
   mqttClient.subscribe(DAS.data);
+  mqttClient.subscribe(`${WirelessModule.base}/#`);
```

Comment:
I just realised that `WirelessModule.all()` doesn't actually return the string used to subscribe to all of the modules. We might want to change that in common at a later time.

Reply:
Would the change have to be in common? If I recall correctly, `topic.js` is a submodule-ish thing.

Reply:
Nah `topics.js` is part of this repo. `topics.py` is in the common repo, but right now that's only a python module with no JS.

Reply:
4565669